src/sagemaker/feature_store/feature_group.py

```diff
@@ -342,15 +343,18 @@ def create(
             create_feature_store_args.update({"online_store_config": online_store_config.to_dict()})
 
         # offline store configuration
-        s3_storage_config = S3StorageConfig(s3_uri=s3_uri)
-        if offline_store_kms_key_id:
-            s3_storage_config.kms_key_id = offline_store_kms_key_id
-        offline_store_config = OfflineStoreConfig(
-            s3_storage_config=s3_storage_config,
-            disable_glue_table_creation=disable_glue_table_creation,
-            data_catalog_config=data_catalog_config,
-        )
-        create_feature_store_args.update({"offline_store_config": offline_store_config.to_dict()})
+        if s3_uri:
```

Comment:
the type differences are a bit odd, since `s3_uri` is expected to be `bool` in some cases.